tools/c7n_tencentcloud/c7n_tencentcloud/resources/cam.py

```diff
@@ -0,0 +1,473 @@
+                               "",
+                               self.resource_manager.config.account_id,
+                               self.resource_type.resource_prefix,
+                               r[self.resource_type.id])
+            qcs_list.append(qcs)
+        return qcs_list
+
+
+@resources.register("cam-user")
+class User(QueryResourceManager):
+    """
+    User - user management in Cloud Access Management (CAM)
+    https://www.tencentcloud.com/document/product/1021/37656
+    """
+    source_mapping = {"describe": DescribeCAM}
+
+    class resource_type(ResourceTypeInfo):
+        """resource_type"""
+        id = "Uin"
+        endpoint = "cam.tencentcloudapi.com"
+        service = "cam"
+        version = "2019-01-16"
+        enum_spec = ("ListUsers", "Response.Data[]", {})
+        taggable = True
+        resource_prefix = "uin"
+        batch_size = 10
+        datetime_fields_format = {
+            "CreateTime": ("%Y-%m-%d %H:%M:%S", pytz.timezone("Asia/Shanghai")),
+            "LastLoginTime": ("%Y-%m-%d %H:%M:%S", pytz.timezone("Asia/Shanghai")),
+            "access_keys.CreateTime": ("%Y-%m-%d %H:%M:%S", pytz.timezone("Asia/Shanghai")),
+            "access_keys.LastUsedDate": ("%Y-%m-%d", pytz.timezone("Asia/Shanghai"))
+        }
+
+    def __init__(self, ctx, data):
+        super().__init__(ctx, data)
+        self._query_client = None
+
+    @property
+    def client(self):
+        if self._query_client is None:
+            self._query_client = self.get_client()
+        return self._query_client
+
+    def augment(self, resources):
+        for item in resources:
+            item["password_enabled"] = not item["ConsoleLogin"] == 0
+            fm = self.resource_type.datetime_fields_format["CreateTime"]
+            item["CreateDate"] = isoformat_datetime_str(item["CreateTime"], fm[0], fm[1])
+        return resources
+
+    def set_user_login_mfa_active(self, resource):
+        """set_user_login_mfa_active"""
+        params = {"SubUin": resource[self.resource_type.id]}
+        resp = self.client.execute_query("DescribeSafeAuthFlagColl", params)
+        mfa_flag = resp["Response"]["LoginFlag"]
+
+        # https://www.tencentcloud.com/document/api/598/32230#LoginActionFlag
+        # several kinds of mfa, mfa is active if any of them is set
+        resource["mfa_active"] = any([v for _, v in mfa_flag.items()])
+
+    def set_user_access_keys(self, resource):
+        """set_user_access_keys"""
+        params = {"TargetUin": resource[self.resource_type.id]}
+        resp = self.client.execute_query("ListAccessKeys", params)
+        access_keys = resp["Response"]["AccessKeys"]
+        if access_keys:
+            # get keys' latest used time
+            for batch in chunks(access_keys, 10):
+                params = {
+                    "SecretIdList": [it["AccessKeyId"] for it in batch]
+                }
+                data = self.client.execute_query("GetSecurityLastUsed", params)
```

Comment:
also better as a dedicated filter.

Reply:
The method is moved to the filter class.
For a dedicated filter, if it has no impact on usage, how about do it when necessary